routes/cart.route.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const router = express.Router();
+const Cart = require("../models/cart.model")
+
+// GET
+router.get('/', async(req ,res)=>{
+
+  try{
+    const prd = await Cart.find()
+    res.json(prd)
+  }catch(err){
+    console.log("err", err)
+  }
+
+});
+
+// POST
+router.post('/', async (req, res)=>{
+  console.log(req.body)
+
+  const addPrd = req.body
+    const newCart = new Cart(
+      addPrd
+      
+    )
+  
+    try {
+      const productData = await newCart.save();
+      res.json(productData);
+    } catch (error) {
+      res.status(400).json({ success: false, message: error });
```

Comment:
instead of sending err send err.message in message like-

`res.status(400).json({ success: false, message: "error while adding product",error:error.message });`

Reply:
Thanks, much better now